packages/studio-base/src/i18n/zh/preferences.ts

```diff
@@ -35,4 +37,11 @@ export const preferences = {
   restartTheAppForChangesToTakeEffect: " 重新启动应用程序以使更改生效。",
   layoutDebugging: "布局调试",
   layoutDebuggingDescription: "显示用于开发和调试布局存储的额外控件。",
+  extensions: "扩展",
+  about: "关于",
+  noExperimentalFeatures: "目前没有实验性的功能。",
+  ros2NativeConnection: "ROS2 的本地连接",
+  ros2NativeConnectionDescription: "启用已废弃的 ROS2 本地连接器",
```

Comment:
```suggestion
  ros2NativeConnection: "ROS 2 的本地连接",
  ros2NativeConnectionDescription: "启用已废弃的 ROS 2 本地连接器",
```

😄 
https://twitter.com/OpenRoboticsOrg/status/1629286675649273856
https://twitter.com/OpenRoboticsOrg/status/1629215777525223424
https://twitter.com/OpenRoboticsOrg/status/1629208251563929600